serverless/cmd/combine_signatures/impl/combine_signatures.go

```diff
@@ -0,0 +1,65 @@
+// Copyright 2021 Google LLC. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// impl provides the implementation of the combine_signatures tool.
+package impl
```

Comment:
I would just straight up put this in a package called `checkpoint(s)`. What's here is a library, not an implementation of the `main` function.

Reply:
Yeah, good call - I was torn between putting it under formats and having it here for now - I've moved it.